packages/mobile/src/screens/search-screen/content/SearchItem.tsx

```diff
@@ -115,7 +115,11 @@ const TrackSearchResult = ({ isLast, item: track }: TrackSearchResultProps) => {
 
   const handlePress = useCallback(() => {
     appendSearchItem(track.title)
-    navigation.push('Track', { id: track.track_id, searchTrack: track })
+    navigation.push('Track', {
+      id: track.track_id,
+      searchTrack: track,
+      canBeUnlisted: false
```

Comment:
tracks in the search results list cannot be unlisted: https://github.com/AudiusProject/audius-client/blob/main/packages/web/src/common/store/search-bar/sagas.ts#L30-L32